web/src/components/CardForm.js

```diff
@@ -0,0 +1,86 @@
+import { Alert, Button, Spinner } from '@cmsgov/design-system-core';
+import PropTypes from 'prop-types';
+import React, { Fragment } from 'react';
+
+const CardForm = ({
+  children,
+  error,
+  legend,
+  onCancel,
+  onSave,
+  primaryButtonText: [primaryButtonNormal, primaryButtonWorking],
+  sectionName,
+  success,
+  title,
+  working
+}) => (
+  <div className="card--container">
+    <div className="ds-l-container">
+      <div className="ds-l-row card">
+        <div className="ds-l-col--1 ds-u-margin-left--auto" />
+        <div className="ds-l-col--12 ds-l-sm-col--10 ds-l-lg-col--6">
+          {!!success && <Alert variation="success">{success}</Alert>}
+          {!!error && <Alert variation="error">{error}</Alert>}
+
+          <h1 className="ds-h1">
+            {sectionName.length > 0 && (
+              <h6 className="ds-h6">{sectionName.toUpperCase()}</h6>
```

Comment:
Calling this out because the dev tools complain about having an `h6` under an `h1`.  But I didn't want to use a different sized header than the class, and I was weary of using a `div` or `p` or something for what is basically a tiny pre-header.

Reply:
👍 To do this semantically within an `h1`, use `span`. That way, a screen reader will pick the whole thing up as a top-level heading and should read it as `Administrator Manage Accounts` or whatever, but we can still style the tagline as not-an-h1.

```suggestion
              <span className="ds-h6 ds-u-display--block">{sectionName.toUpperCase()}</span>
```

Reply:
`display: block` because spans are inline elements, yadda yadda